keyboards/beekeeb/piantor/keymaps/vial/config.h

```diff
@@ -0,0 +1,8 @@
+/* keyboard uid */
+#define VIAL_KEYBOARD_UID {0xDC, 0x6D, 0x9C, 0x42, 0x53, 0x42, 0x75, 0x0F}
```

Comment:
@lesshonor is this necessary since vendorId and productId are define in vial.json?